testing/trino-product-tests-launcher/src/main/resources/docker/presto-product-tests/conf/environment/multinode-clickhouse/clickhouse.properties

```diff
@@ -0,0 +1,4 @@
+connector.name=clickhouse
+connection-url=jdbc:clickhouse://clickhouse:8124/
```

Comment:
Since your containers are named `clickhouse-1`, `clickhouse-2` and so on you should use `clickhouse-1` here.
(Docker will ensure that hostnames (= container names) get resolved across containers correctly - no need to edit /etc/hosts)

Reply:
Thanks for the clarification, it also means I need to undo the changes in `README.md`.